Decisions.Equifax/ConsumerCreditReport/ConsumerCreditReportSteps.cs

```diff
@@ -1,18 +1,8 @@
-﻿using System;
-using System.IO;
-using System.Net;
-using System.Text.Encodings.Web;
-using Decisions.Equifax.ConsumerCreditReport.Request;
+﻿using Decisions.Equifax.ConsumerCreditReport.Request;
 using Decisions.Equifax.ConsumerCreditReport.Response;
 using DecisionsFramework;
-using DecisionsFramework.Design.ConfigurationStorage.Attributes;
 using DecisionsFramework.Design.Flow;
-using DecisionsFramework.Design.Flow.CoreSteps;
-using DecisionsFramework.Design.Flow.Mapping;
-using DecisionsFramework.Design.Properties.Attributes;
 using DecisionsFramework.ServiceLayer;
-using DecisionsFramework.ServiceLayer.Services.ContextData;
-using Newtonsoft.Json;
 
 namespace Decisions.Equifax.ConsumerCreditReport
 {
```

Comment:
Can't comment on unchanged lines but Line 12 is unnecessary since any logging calls were moved to the utility class.